src/co/edu/uptc/model/Movie.java

```diff
@@ -1,14 +1,23 @@
 package co.edu.uptc.model;
 
+import java.lang.reflect.Array;
+import java.util.ArrayList;
 import java.util.Date;
 
 public class Movie extends Multimedia {
     private int duration;
+    private ArrayList<Category> categories;
 
-    public Movie(int id, String title, String synopsis, Date releaseDate) {
+    public Movie() {
+
+    }
+
+    public Movie(int id, String title, String synopsis, Date releaseDate, ArrayList<Category> categories) {
         super(id, title, synopsis, releaseDate);
         this.duration = duration;
+        this.categories = new ArrayList<>();
```

Comment:
En este caso, si se va a pasar por parametro al constructor es necesario que el código sea así:
`
this.categories = categories;
`